### PR TITLE
New Port: Super Meat Boy

### PR DESCRIPTION
**Pull request by fexofenadine**
# New Port for Super Meat Boy

## Game Information
- **Title**: Super Meat Boy
- **URL**: [Super Meat Boy Steam page](https://store.steampowered.com/app/40800/Super_Meat_Boy/)

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

**Comment by fexofenadine** [comment]
currently testing in [Discord: Testing-n-Dev - SMB thread](https://discord.com/channels/1122861252088172575/1202446614992650280)

**Comment by fexofenadine** [comment]
Updated with current test build

File: screenshot.jpg
Comment (kloptops) [inline screenshot.jpg:1]: Just need to delete this file. 😁

**Review by kloptops — CHANGES_REQUESTED** [review]
Just need to delete the screenshot at the root of the repo.

**Review by fexofenadine — COMMENTED** [review]
done!

**Review by kloptops — APPROVED** [review]
👍

**Review by tabreturn — APPROVED** [review]

**Pull request merged**